mastracode/src/acp/index.ts

```
@@ -0,0 +1,74 @@
+import type { HarnessMode } from '@mastra/core/harness';
+
+import { createMastraCode } from '../index.js';
+import { releaseAllThreadLocks } from '../utils/thread-lock.js';
+import { runAcpServer } from './server.js';
+import { setAutoApprove } from './event-mapper.js';
```

ESLint

[error] `./event-mapper.js` import should occur before import of `./server.js`

(import/order)